test/expectation/.expected

```diff
@@ -4,5 +4,6 @@ launch-darkly.key=secret_value
 msk.endpoint=https://msk.com
 openai.key=beepboop
 private=password
+redis.kerberos-authz.url=kerberos-authz.cache.dev.pleo.io
```

Comment:
Should we add an entry to test for arrays too, like what jens wants to do? 🤔

Reply:
it works 💣

Reply:
Quality software